production/resources/js/sm-share-sidebar.js

```diff
@@ -0,0 +1,60 @@
+// Find and set page url, title, and summary
```

Comment:
Same thing here, this js cleanup should be a separate branch. You'll get the tasks done much faster by focusing on one piece at a time and the reviews will be much faster as well. You're also likely to avoid conflicts.